crates/proto/src/op/header.rs

```diff
@@ -7,6 +7,8 @@
 
 //! Message metadata
 
+#![allow(clippy::bool_to_int_with_if)]
```

Comment:
I think this should go in `lib.rs`, so we only need it once.

Reply:
Sure will do it tonight I was I dire situation searching the good place.

Reply:
done!